src/services/utilities.ts

```diff
@@ -656,4 +656,34 @@ module ts {
     export function isJavaScript(fileName: string) {
         return fileExtensionIs(fileName, ".js");
     }
+
+    export function getDeclaredName(typeChecker: TypeChecker, symbol: Symbol, location: Node) {
+        // If this is an export or import specifier it could have been renamed using the 'as' syntax.
+        // If so we want to search for whatever is under the cursor.
+        if (isImportOrExportSpecifierName(location)) {

```

Comment:
as per our discussion we need to remove this, and put it in getSymbolInfo instead. and this way getInternedName is just symbol.name all the time and we can remove it.